File: BlogService/src/main/java/net/blueshell/blogservice/BlogService.java
package net.blueshell.blogservice;

import net.blueshell.db.BaseModelService;
import org.springframework.beans.factory.annotation.Autowired;

import java.util.UUID;

@org.springframework.stereotype.Service
public class BlogService extends BaseModelService<Blog, UUID, BlogRepository> {

    @Autowired
    public BlogService(BlogRepository blogRepository) {
        super(blogRepository);
    }
}

File: api/src/main/kotlin/net/blueshell/api/blog/application/BlogService.kt
package net.blueshell.api.blog.application

import net.blueshell.api.blog.persistence.Blog
import net.blueshell.api.blog.persistence.repository.BlogRepository
import net.blueshell.api.shared.service.BaseModelService
import org.springframework.beans.factory.annotation.Autowired
import org.springframework.context.ApplicationEventPublisher
import org.springframework.stereotype.Service

@Service
class BlogService @Autowired constructor(blogRepository: BlogRepository, events: ApplicationEventPublisher) :
    BaseModelService<Blog, Long, BlogRepository>(blogRepository)
